xero_files.yaml

```diff
@@ -791,7 +793,7 @@ paths:
         '204':
           description: Successful deletion - return response 204 no content
           x-isEmpty: true
-  /Inbox:
+/Inbox:
```

Comment:
Invalid indentation. please indent the endpoints at the same level like the previous endpoints.

Reply:
This is done  at the same level like the previous endpoints.